tox.ini

```diff
@@ -160,7 +160,7 @@ envlist =
     python-framework_tornado-{py38,py39,py310,py311}-tornadomaster,
     rabbitmq-messagebroker_pika-{py27,py37,py38,py39,pypy27,pypy38}-pika0.13,
     rabbitmq-messagebroker_pika-{py37,py38,py39,py310,py311,pypy38}-pikalatest,
-    kafka-messagebroker_confluentkafka-{py27,py37,py38,py39,py310,py311}-confluentkafkalatest,
+    kafka-messagebroker_confluentkafka-{py37,py38,py39,py310,py311}-confluentkafkalatest,
```

Comment:
Why are we dropping 2.7 testing here? It looks like it's still supported on the latest version.

Reply:
From my testing the library is broken on Python 2 now, it throws import errors on the latest versions.

Reply:
Oh ouch ok. Sounds good then!